cmd/recover/recover.go

```diff
@@ -60,3 +65,33 @@ func recoverCommandF(chaos *chaosd.Server, options *recoverCommand) {
 
 	utils.NormalExit(fmt.Sprintf("Recover %s successfully", options.uid))
 }
+
+func completeUid(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+	completionCtx := newCompletionCtx()
+	completionDep := fx.Options(
+		server.Module,
+		fx.Provide(func() *completionContext {
+			return completionCtx
+		}),
+	)
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	go func() {
+		if err := utils.FxNewAppWithoutLog(completionDep, fx.Invoke(listUid)).Start(ctx); err != nil {
```

Comment:
how about only listing `uid` which are `success` status? only `success` experiment need to recover

Reply:
Reasonable

Reply:
Done